lib/daru/io/io.rb

```diff
@@ -221,6 +227,25 @@ def from_csv_hash(path, opts)
 
         headers.each_with_index.map { |h, i| [h, csv_as_arrays[i]] }.to_h
       end
+
+      def from_html_get_list(page)
+        page.search('table').map { |table|
+          index = table.search('th').map { |header| header.text.strip }
+          data = table.search('tr').map { |row| row.search('td').map { |val| val.text.strip } }.keep_if { |a| a!=[] }
+          {'index' => index, 'data' => data}
+        }.keep_if { |a| a['index'] != [] }
+      end
+
+      def from_html_get_df_list(list)
+        list.map { |ele|
+          begin
+            indexes, data = ele['index'], ele['data'].transpose
+            hash = Hash[indexes.map { |index| [index,data[indexes.index(index)]] }]
+            Daru::DataFrame.new(hash)
+          rescue # rubocop:disable Lint/HandleExceptions
```

Comment:
Wikipedia has many tables (most of their links / navigation menus are also tables), and most of them don't have structured data of much importance. Hence, to differentiate the "useful" tables from the "other" tables, I had followed this `table > th` , `table > tr > td` node-paths. In case of the "other" tables, trying to access these node-paths will throw a variety of error(s) depending on the node structure, like `nil NilClass`.

So, I had to `rescue` in order to get the list of `useful` tables as `Daru::Dataframe`s. However, a simple `rescue` block with no code for exception shows a Rubocop offense. Hence, I had to rescue, and disable rubocop for this line.